### PR TITLE
Fixed code sign error message width

### LINKED ISSUE #11917: Output of Flutter tool, when iOS can't be built, doesn't fit 80 columns
Terminals seem to default to 80 chars wide, however the error message for when flutter tool can't build the iOS app is wider the 80 chars:

screen shot 2017-09-03 at 2 10 09 pm

### PR DESCRIPTION
**Pull request by jslavitz**
fixed https://github.com/flutter/flutter/issues/11917 . Also fixed the other error messages that were over 80 columns

### CODE REVIEW

**Comment by jonahwilliams** [comment]
@jslavitz , since this change is in flutter_tools you need to run the tests there separately.  For example, test/ios/mac_test.dart is failing since it just checks that the string is the same as a golden

File: packages/flutter/test/material/snack_bar_test.dart [hunk 1]
```diff
@@ -482,4 +482,49 @@ void main() {
...
+                          width: 100.0,
+                          key: tapTarget
+                      )
+                  );
+                }
+            )
+        )
+    ));
+
+    expect(find.text('bar1'), findsNothing);
```
Comment (jonahwilliams) [inline packages/flutter/test/material/snack_bar_test.dart]: This doesn't really test the default duration.  You'll want to use
```
tester.pump(duration)
```
to verify the timing - see https://docs.flutter.io/flutter/flutter_test/WidgetTester/pump.html

Comment (jonahwilliams) [reply packages/flutter/test/material/snack_bar_test.dart]: also this is in the wrong PR 📦

**Review by jonahwilliams — APPROVED** [review]
LGTM to submit on green

**Comment by jslavitz** [comment]
@Hixie - Does this look good to submit?

**Comment by Hixie** [comment]
Sure, we can land this. Thanks for your contribution!

fyi @gspencergoog

**Pull request merged**